src/node/desktop/test/unit/main/browser-window.test.ts

```diff
@@ -54,12 +46,10 @@ describe('BrowserWindow', () => {
       });
     }
 
-    await win.loadURL('http://localhost:9876');
-    server.kill();
+    await win.loadURL('http://localhost:9875');
+    server.close();
```

Comment:
what's the reason for changing from kill to close here? is it just unnecessary?

Reply:
`kill` is when `server` was a child process object. It's now a node server object, which `close` will gracefully shut down the server.